src/site/_filters/find-by-url.js

```diff
@@ -33,11 +30,6 @@ let memo;
  * @see {@link https://github.com/11ty/eleventy/issues/399}
  */
 const memoize = (collection) => {
```

Comment:
The warning that was previously here was essentially noise (it logged every time you edited a file in dev mode) so I just removed it.